docs/pages/2020_BIGMAP/sections/browse.rst

```diff
@@ -0,0 +1,66 @@
+.. _BIGMAP_2020_browse:
+
+Browsing your database
+======================
+
+This final section of the tutorial is optional, and has the purpose of introducing a couple of ways in which you can browse an AiiDA database.
+
+There are two main reasons you may want to browse a database.
+The first one is to get a general idea of the content of it, for which the explore tool of the `Materials Cloud <https://www.materialscloud.org/explore/menu>`_ can be of great use.
+The second one is a more specific search with the purpose of gathering more particular and well defined results.
+For this second use, we will reccommend direct queries via the ``QueryBuilder``.
+
+.. note::
+
+  Although we introduce them for different purposes, both of these tools can ultimately be used for general exploration or more specific searches.
+  Even more, proper knowledge of how to use of the querybuilder may also be essential when creating running scripts and even workflows.
+
+Exploring the database
+----------------------
+
+Placeholder for the future content.
+
+
+Querying the database
+---------------------
+
+Copy content here. The note could probably go next to the wget of the database (maybe that part will go before the "exploring" section above).
+
+.. note::
+
+    When perfoming calculations for a publication, you can export your provenance graph (meaning all the content of the nodes and their connections) into an archive file using ``verdi export create``, and then upload it to the `Materials Cloud Archive`_, enabling your peers to explore the provenance of your calculations online.
+
+
+What next?
+----------
+
+You now have a first taste of the type of problems AiiDA tries to solve.
+Here are some options for how to continue:
+
+* Continue with the :ref:`in-depth tutorial<2020_Intro_Week_Homepage>`.
+* Download the `Quantum Mobile`_ virtual machine and try running the tutorial on your laptop instead.
+* Try `setting up AiiDA`_ directly on your laptop.
+
+.. dropdown:: Remember your workchain!
```

Comment:
Do we need to duplicate this? It's already in the "Finishing the workchain" section, no? 🙃

Reply:
Yeah, you are right, probably no. I'll just leave a sentence as a reminder and maybe a link to the previous section.